streamer/src/quic.rs

```diff
@@ -76,6 +77,7 @@ pub(crate) fn configure_server(
     server_tls_config.alpn_protocols = vec![ALPN_TPU_PROTOCOL_ID.to_vec()];
 
     let mut server_config = ServerConfig::with_crypto(Arc::new(server_tls_config));
+    server_config.concurrent_connections(max_concurrent_connections);
```

Comment:
If the server is already at capacity and rate of incoming connections is larger than how fast the application layer can prune extra connections, then I would guess this will stay at capacity; i.e. a dropped connection is immediately replaced by a new connection.

In that case which connections make it through is entirely decided by quinn which is stake agnostic. In other words an unstaked clients can front run staked node by spamming and preventing stake based priority (which in the application layer and not quinn) be enforced.

Is that correct?

Reply:
Yes, that is why we left 25% buffer so that in normal case the app layer can still make decisions in its desired range. This change enable Quinn to more efficiently throttle spamming connections.